crates/holochain_sqlite/src/schema.rs

```diff
@@ -29,7 +29,14 @@ pub static SCHEMA_CELL: Lazy<Schema> = Lazy::new(|| Schema {
 });
 
 pub static SCHEMA_CONDUCTOR: Lazy<Schema> = Lazy::new(|| Schema {
-    migrations: vec![M::initial(include_str!("sql/conductor/schema/0.sql"))],
+    migrations: vec![
+        M::initial(include_str!("sql/conductor/schema/0.sql")),
+        // Everything in schema 0 has IF NOT EXISTS on it so we can rerun it.
+        M {
+            forward: include_str!("sql/conductor/schema/0.sql").into(),
+            _schema: "".into(),
+        },
+    ],
```

Comment:
@maackle i think that's something to discuss with @neonphog who was suggesting we use `IF NOT EXISTS` quite generally, i suppose we can put them all in their own files and it would still have that effect but we probably need to figure out what our exact approach is